api/challenge_api/app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
```

Comment:
I'm unfamiliar with ActionMailer, what role do you envision this class will play in the app?

Reply:
This would be relevant if our application needed to send e-mail -- it would do so with HTML so that explains the HTML we don't need which is currently living in layouts. As far as I know we won't need to send email, so I'll remove this along with the associated views